### PR TITLE
Feat install app

### PR DESCRIPTION
**Pull request by frdomovic**
# Feat install app

## Summary:
Add install application page and update api call to match updated node api.

### CODE REVIEW

File: src/api/dataSource/NodeDataSource.ts [hunk 1]
```diff
@@ -407,9 +407,10 @@ export class NodeDataSource implements NodeApi {
         await this.client.post<InstallApplicationResponse>(
           `${getAppEndpointKey()}/admin-api/install-application`,
           {
-            contract_app_id: selectedPackageId,
-            version: selectedVersion,
             url: ipfsPath,
+            version: selectedVersion,
+            // TODO: parse hash to format
```
Comment (MatejVukosav) [inline src/api/dataSource/NodeDataSource.ts:412]: Why is this required?

**Review by MatejVukosav — APPROVED** [review]

File: src/api/dataSource/NodeDataSource.ts (hunk 1 above)
Comment (frdomovic) [reply src/api/dataSource/NodeDataSource.ts:412]: check is downloaded hash is the same as saved in package manager. Preventing installation of corrupt or changed version
